### DISCUSSION TITLE
Flags are not displayed in production

### DISCUSSION THREAD

**Discussion by ArtMin96** [ysfkaya/filament-phone-input#25] [Q&A]
Flags are not displayed in production.
**GET https://********/phone-input-flags.png 404 (Not Found)**
![image](https://github.com/ysfkaya/filament-phone-input/assets/29732308/bc3d8d02-2cea-4351-8f6e-40bf29cb35d1)

**Comment by ysfkaya** [comment]
Are you using it in tenants?

**Reply by ArtMin96** [reply]
No

**Reply by ysfkaya** [reply]
I need more detail about the error.

What's the version of the package?
How did you deploy the app to production server. E.g cached routes
How was it used? Are you using outside from Filament?
